### DISCUSSION TITLE
macos部署 alist 时 ， 如何不上传 .DS_Store 文件？

### DISCUSSION THREAD

**Discussion by bbbbio** [AlistGo/alist#6257] [General]
云盘中会被上传大量的.DS_Store ，必须手动删除
![截屏2024-03-23 11 56 40](https://github.com/alist-org/alist/assets/73514862/576223f0-68ba-4824-ba2e-c0de4aea2065)

**Comment by bbbbio** [comment]
<img width="711" alt="image" src="https://github.com/alist-org/alist/assets/73514862/2d256cd0-0581-42c0-8376-10c08758c3ae">
每操作一次就会越来越多，看着非常的痛苦

**Comment by pongfcnkl** [comment]
看你挂载软件，比如rclone就有排除参数--exclude

**Comment by ygyGitHub** [comment]
这个问题还没解决呢么，用访达挂载的，问题很严重
